README.md

```diff
@@ -179,6 +181,11 @@ Follow instructions from the link below to build and run tests:
 * SYCL\* 2020 specification:
 [https://www.khronos.org/registry/SYCL/](https://www.khronos.org/registry/SYCL/)
 * More information on oneAPI and DPC++ is available at [https://www.oneapi.com/](https://www.oneapi.com/)
+* Check the following documentation for guidance on using the migrated SYCL code with non-Intel devices using the DPC++ Compatibility Tool:
+
+  * [Targeting nVIDIA GPUs](https://developer.codeplay.com/products/oneapi/nvidia/2023.2.1/guides/get-started-guide-nvidia#use-dpc-to-target-nvidia-gpus)
+
+  * [Targeting AMD GPUs](https://developer.codeplay.com/products/oneapi/amd/2023.2.1/guides/get-started-guide-amd#use-dpc-to-target-amd-gpus)
```

Comment:
https://developer.codeplay.com/products/oneapi/amd/home/  use this URL instead.

Reply:
Changed to this URL. Done.